compare_anagrams/vishS/anagram.js

```diff
@@ -0,0 +1,12 @@
+function checkAnagram(string1, string2){
+    function stringToArray(str){
+        str = str.split("").sort().join().trim();   
+        return str;       
+    }    
+    if(string1.length == string2.length && stringToArray(string1) === stringToArray(string2)) {
+     return true;   
```

Comment:
you should indent this return statement by another 3 spaces to be consistent with the rest of the file.